config/routes.rb

```diff
@@ -2,10 +2,19 @@
 
 Rails.application.routes.draw do
   root 'users#index'
-
+  
   get '/users', to: 'users#index'
   get '/users/:user_id', to: 'users#show'
-
+  
   get '/users/:user_id/posts', to: 'user_posts#index'
+  get '/users/:user_id/posts/new', to: 'user_posts#new'
   get '/users/:user_id/posts/:post_id', to: 'user_posts#show'
+  get '/users/:user_id/posts/:post_id/new_comment', to: 'comments#createcomment'
+
+  
+  post '/users/:user_id/posts', to: 'user_posts#create'
+  post '/users/:user_id/posts/:post_id/new_comment', to: 'comments#submitcomment'
+  post '/users/:user_id/posts/:post_id/new_like', to: 'likes#submitlike'
```

Comment:
The user is unable to like a post. Please see the error below which occurs because of the wrong controller in route.
![errors](https://user-images.githubusercontent.com/35504149/155739075-2f6d4941-fd03-4dfa-a077-8240e5b8d1c0.png)

Reply:
@SajjadAhmad14 hello are u using the latest version of this PR?

Reply:
Hi @RafaelEchart so now I see the problem. I cloned the project and meanwhile, you made a change so it was confusing. Your project is working fine now.